shared/tasks/assets.mjs

```diff
@@ -19,7 +20,13 @@ export async function write (filePath, result) {
   const code = 'css' in result ? result.css : result.code
 
   // 1. Write code (example.js)
-  writeTasks.push(writeFile(filePath, code))
+  writeTasks.push(files.write(base, {
```

Comment:
Our tasks had two ways to write to disk

1. Shared assets task
2. Shared files task

Testing in https://github.com/alphagov/govuk-frontend/pull/3498 found we forgot to run `mkdir()` (to make new directories) before writing to disk 😮

This only became obvious once `package/dist` was deleted so both files now share `files.write()`